src/index.js

```diff
@@ -69,10 +69,10 @@ ${unused.join(`\n`)}`);
   }
 
   _getFileDepsMap(compilation) {
-    const fileDepsBy = compilation.fileDependencies.reduce((acc, usedFilepath) => ({
-      ...acc,
-      [usedFilepath]: usedFilepath,
-    }), {});
+    const fileDepsBy = compilation.fileDependencies.reduce((acc, usedFilepath) => {
```

Comment:
It might be worth using `Set` if only to avoid the eslint override :-) but it'd be great to try it and determine if it's significantly better.